api/proto/teleport/legacy/types/webauthn/webauthn.proto

```diff
@@ -56,6 +56,36 @@ message SessionData {
   // "required".
   // An empty value is treated equivalently to "discouraged".
   string user_verification = 5 [(gogoproto.jsontag) = "userVerification,omitempty"];
+  // Scope authorized by this webauthn session.
+  ChallengeScope scope = 6 [(gogoproto.jsontag) = "scope,omitempty"];
```

Comment:
In a similar vein to the above, I think I'd rather define those messages outside of webauthn, as we are growing past the spec in features. We could do that by making SessionData a storage-only / implementation only type.

Reply:
Is there a reason for SessionData to have a proto definition? It seems at first glance like all of `webauthn.proto` could just be internal types.

Reply:
We do use the assertion/credential messages in RPCs, and consequently everything contained in them. This amounts to all types in here but SessionData, which could have been a Go struct.